txs/store.go

```diff
@@ -0,0 +1,86 @@
+package txs
+
+import (
+	"time"
+
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/log"
+	"github.com/spacemeshos/go-spacemesh/sql"
+	"github.com/spacemeshos/go-spacemesh/sql/transactions"
+)
+
+type store struct {
+	logger log.Log
+	db     *sql.Database
+}
+
+func newStore(db *sql.Database, logger log.Log) *store {
+	return &store{
+		logger: logger,
+		db:     db,
+	}
+}
+
+func (s *store) add(tx *types.Transaction, received time.Time) error {
+	return nil
+}
+
+func (s *store) has(tid types.TransactionID) bool {
+	has, err := transactions.Has(s.db, tid)
+	return err == nil && has
```

Comment:
i don't think that we should ignore db errors. on any unexpected db error app should not continue normal flow

Reply:
fixed